Makefile

```diff
@@ -74,6 +74,25 @@ $(TOOLS)/gojq: PACKAGE=github.com/itchyny/gojq/cmd/gojq
 .PHONY: tools
 tools: $(CROSSLINK) $(DBOTCONF) $(GOLANGCI_LINT) $(MISSPELL) $(GOCOVMERGE) $(STRINGER) $(PORTO) $(GOJQ) $(SEMCONVGEN) $(MULTIMOD) $(SEMCONVKIT)
 
+# Python tools
+
+VENVDIR = $(CURDIR)/venv
+PYTOOLS = $(VENVDIR)/bin
+PY = $(PYTOOLS)/python
+
+# Create a virtual environment for Python tools.
+$(PYTOOLS):
+# Check if python3 is available.
+	@python3 --version >/dev/null 2>&1 || (echo "python3 is not available. Please install it." && exit 1)
```

Comment:
Maybe using a Docker container?

For reference [here](https://github.com/streetsidesoftware/cspell-cli/pull/183) is what I did for [`cspell`](https://github.com/streetsidesoftware/cspell) so that it could be adopted for non-JS repositories.

Reply:
I agree. I will work on a new version using a docker version. Thanks for the reviews @MrAlias and @pellared.

Reply:
Updated PR which should have resolved this conversation.